heron/spi/src/java/com/twitter/heron/spi/common/keys.yaml

```diff
@@ -62,6 +62,12 @@ SCHEDULER_IS_SERVICE:                heron.scheduler.is.service
 SCHEDULER_JAR:                       heron.jars.scheduler
 
 ################################################################################
+# keys for Topology Update command
+################################################################################
+TOPOLOGY_UPDATE_DEACTIVATE_SECONDS:  heron.topology.update.deactivate.seconds
+TOPOLOGY_UPDATE_REACTIVATE_SECONDS:  heron.topology.update.reactivate.seconds
```

Comment:
@ashvina implementing callbacks would be great but that's a much more involved feature that we'd need to think through and ideally have a solid use case for before building it. Currently we don't support any network callbacks to other systems so to do so we'd need to design the protocol, the register/deregister callback mechanisms, etc. I suggest we look into that once we have a solid use case for it's use.